src/RepoUtil/project.json

```diff
@@ -0,0 +1,21 @@
+{
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "version": "1.0.0-rc2-3002702",
+      "type": "platform"
```

Comment:
Since we aren't using the cli so this doesn't really make it a .NET Core app. The runtime config and the fact we use dotnet.exe to run it makes it a .NET Core app.

We should have another project that builds this for desktop so we can also run it there.